index.html

```diff
@@ -215,7 +218,52 @@ <h5 class="mb">Would you like to know more information on bentos around you?</h5
 		        'opacity': 1 - (scrollPos / 250)
 		    });
 		}
+
+		// Check if element is on screen or not
+
+		$.fn.isOnScreen = function() {
+			var viewport = {};
+		    viewport.top = $(window).scrollTop();
+		    viewport.bottom = viewport.top + $(window).height();
+		    var bounds = {};
+		    bounds.top = this.offset().top;
+		    bounds.bottom = bounds.top + this.outerHeight();
+		    return ((bounds.top <= viewport.bottom) && (bounds.bottom >= viewport.top));
+		};
+		
+		Math.easeOutQuad = function (t, b, c, d) {
+			t /= d;
+			return -c * t*(t-2) + b;
+		};
+
+		$.fn.countUpAnimate = function(start, target) {
+			var elements	= this;
+			var clr 		= null;
+			// Max distant is 100
+			var start		= target - start > 100 ? target - 100 : start;
+			var count 		= start;
+			var elapsedTime = 0;
+
+			(function () {
+			    var inloop;
+			    clearTimeout(clr);
+			   (inloop = function () {
+			   		if (count >= target) {
+			            return;
+			        }
+			        elapsedTime += 1;
+			        count = Math.round(Math.easeOutQuad(elapsedTime, start, target - start, target));
+			   		count = count  > target ? target : count; 
+			        elements.text(count);
+			        clr = setTimeout(inloop, 32);
```

Comment:
Never mind found it 
http://stackoverflow.com/questions/729921/settimeout-or-setinterval